webpack.config.js

```diff
@@ -0,0 +1,41 @@
+module.exports = {
+  module: {
+    rules: [{
+      test: /\.jsx?$/,
+      exclude: /node_modules\/(?!@terrestris)/,
```

Comment:
This should not be necessary anymore, with latest versions of `@terrestris` packages. They now also export es5 builds.

Reply:
Edit: I don't see any dependencies to `@terrestris` packages in your package.json. So you should check if you have to exclude the whole `/node_modules` directory or if you simply can remove the whole line.

Reply:
I'm leaving that in since in my experience this works just fine if we ever add any terrestris dependencies. We're also using `d3` es6 imports.